qiskit/test/mock.py

```diff
@@ -233,26 +233,31 @@ def __init__(self):
             pulse_library=[PulseLibraryItem(name='test_pulse_1', samples=[0.j, 0.1j]),
                            PulseLibraryItem(name='test_pulse_2', samples=[0.j, 0.1j, 1j]),
                            PulseLibraryItem(name='test_pulse_3',
-                                            samples=[0.j, 0.1j, 1j, 0.5 + 0j])],
+                                            samples=[0.j, 0.1j, 1j, 0.5 + 0j]),
+                           PulseLibraryItem(name='test_pulse_4',
+                                            samples=7*[0.j, 0.1j, 1j, 0.5 + 0j])],
             cmd_def=[Command(name='u1', qubits=[0],
                              sequence=[PulseQobjInstruction(name='fc', ch='d0',
                                                             t0=0, phase='-P1*np.pi')]),
                      Command(name='u3', qubits=[0],
```

Comment:
Could we also augment the `u3`/`u2`/`u1` to have similar parameters to what is returned by the backend `P0`,`P1`,`P2` for better use-case test coverage?